Library/Homebrew/github_packages.rb

```diff
@@ -194,7 +201,18 @@ def validate_schema!(schema_uri, json)
     exit 1
   end
 
-  def upload_bottle(user, token, skopeo, formula_full_name, bottle_hash, dry_run:, warn_on_error:)
+  def download(user, token, skopeo, image_uri, root, dry_run:)
+    puts
+    args = ["copy", "--all", image_uri.to_s, "oci:#{root}"]
+    if dry_run
+      puts "#{skopeo} #{args.join(" ")} --src-creds=#{user}:$HOMEBREW_GITHUB_PACKAGES_TOKEN"
```

Comment:
Not saying we should but: could moving this skopeo with or without dry run logic into a method (as it's used three times now)

Reply:
The different args, even ones like `print_stderr` make it a bit tricky.

Reply:
Fine with me 👍🏻